lib/ui/semantics.dart

```diff
@@ -743,8 +743,11 @@ class SemanticsUpdateBuilder extends NativeFieldWrapperClass2 {
     required double thickness,
     required Rect rect,
     required String label,
+    List<Map<String, dynamic>>? labelAttributes,
```

Comment:
Also commented on the framework side: wondering whether each attribute has to be represented as a map or if it could just be a list where we know that index 1 is the start, index 2 is the end, index 3 is the type, etc.